src/conditionals/programs_with_decision_points/hard/q15/MainTest.java

```diff
@@ -0,0 +1,68 @@
+package conditionals.programs_with_decision_points.hard.q15;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+    
+    @Override
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "1\n1";
+        
+        return new Clause[]{
+                new StringLiteral("Enter num1 "),
```

Comment:
since you included the optional, the extra space from the end of the two basic string literals should be removed